onnxruntime/contrib_ops/cpu/quantization/dynamic_quantize_matmul.cc

```diff
@@ -161,6 +161,127 @@ class DynamicQuantizeMatMul final : public MatMulIntegerToFloatBase {
 
   Status Compute(OpKernelContext* context) const override;
 
+  Status PrePack(const Tensor& tensor, int input_idx, AllocatorPtr alloc,
+                 /*out*/ bool& is_packed,
+                 /*out*/ PrePackedWeights* prepacked_weights) override {
+    // only pack Matrix B
+    if (input_idx == GetBIdx()) {
+      const Tensor* b_zp_tensor{nullptr};
+      bool b_zp_present = false;
```

Comment:
Little confused with this initialialization and the logic below when zero point is a constant input.

What happens when zp is present and not a constant ? We should gate usage of KleidiAI appropriately in that scenario ?